$OpenBSD: patch-lib_net_c,v 1.3 2014/10/07 09:55:09 kirby Exp $

evutil.h already included in net.h
Merged upstream 620a6aaf3095dc467f88a4f049d526eeca39cd61

--- lib/net.c.orig	Tue Jan 21 12:45:00 2014
+++ lib/net.c	Tue Jan 21 12:46:08 2014
@@ -32,8 +32,6 @@
 
 #include <fcntl.h>
 
-#include <evutil.h>
-
 #include "net.h"
 
 
